domain/status/service/service.go

```diff
@@ -594,6 +601,38 @@ func (s *Service) DeleteUnitPresence(ctx context.Context, unitName coreunit.Name
 	return s.st.DeleteUnitPresence(ctx, unitName)
 }
 
+// CheckUnitStatusesReadyForMigration returns true is the statuses of all units
```

Comment:
This function just returns an error, not a boolean.

Reply:
Good spot! This function did originally return a bool, but not any more